torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -0,0 +1,265 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+from torch.nn import Conv2d
+from torch.nn import init
+from torch.nn._intrinsic import ConvBn2d as NNConvBn2d
+from torch.nn._intrinsic import ConvBnReLU2d as NNConvBnReLU2d
+from torch.nn._intrinsic import ConvReLU2d as NNConvReLU2d
+from ....qat.modules.conv import Conv2d as QATConv2d
+from torch.nn import Parameter
+import torch.nn.functional as F
+
+
+class ConvBn2d(Conv2d):
+    r"""
+    A ConvBn2d module is a module fused from Conv2d and BatchNorm2d,
+    attached with FakeQuantize modules for both output activation and weight,
+    used in quantization aware training.
+
+    We combined the interface of :class:`torch.nn.Conv2d` and
+    :class:`torch.nn.BatchNorm2d`.
+
+    Implementation details: https://arxiv.org/pdf/1806.08342.pdf section 3.2.2
+
+    Similar to :class:`torch.nn.Conv2d`, with FakeQuantize modules initialized
+    to default.
+
+    Attributes:
+        freeze_bn:
+        observer: fake quant module for output activation, it's called observer
+            to align with post training flow
+        weight_fake_quant: fake quant module for weight
+
+    """
+    __FLOAT_MODULE = NNConvBn2d
+
+    def __init__(self,
+                 # conv2d args
+                 in_channels, out_channels, kernel_size, stride=1,
+                 padding=0, dilation=1, groups=1,
+                 bias=True, padding_mode='zeros',
+                 # bn args
+                 # num_features: enforce this matches out_channels before fusion
+                 eps=1e-05, momentum=0.1,
+                 # affine: enforce this is True before fusion?
```

Comment:
Please remove comments

Reply:
I think we want to keep these for documentation purpose?

Reply:
Sure, please remove the comments with question marks like #affine etc